### DISCUSSION TITLE
使用docker部署和宿主机方式部署哪种好呢？

### DISCUSSION THREAD

**Discussion by a3225167** [apache/doris#15135] [A - General / Q&A] [answered]
突然发现官方文档增加了docker部署的方式，那么哪种方式部署比较好呢？ 包括后续版本升级啥的。
<img width="303" alt="image" src="https://user-images.githubusercontent.com/32891043/208061877-b3fa6b70-416a-40b9-9064-4fac055af354.png">

**Comment by zy-kkk** [comment, marked as answer]
宿主机部署，docker是为了快速试用，不适合生产部署
